lib/blocks.php

```diff
@@ -397,299 +357,3 @@ function gutenberg_register_legacy_social_link_blocks() {
 }
 
 add_action( 'init', 'gutenberg_register_legacy_social_link_blocks' );
-
-/**
- * Checks whether the current block type supports the feature requested.
- *
- * @param WP_Block_Type $block_type Block type to check for support.
- * @param array         $feature    Path of the feature to check support for.
- * @param mixed         $default    Fallback value for feature support, defaults to false.
- *
- * @return boolean                  Whether or not the feature is supported.
- */
-function gutenberg_block_has_support( $block_type, $feature, $default = false ) {
-	$block_support = $default;
-	if ( $block_type && property_exists( $block_type, 'supports' ) ) {
-		$block_support = _wp_array_get( $block_type->supports, $feature, $default );
-	}
-
-	return true === $block_support || is_array( $block_support );
-}
-
-/**
- * Updates the shape of supports for declaring fontSize and lineHeight.
- *
- * @param array $metadata Metadata for registering a block type.
- * @return array          Metadata for registering a block type with the supports shape updated.
- */
-function gutenberg_migrate_old_typography_shape( $metadata ) {
```

Comment:
Added to WordPress core in 5.8.0 release:
https://github.com/WordPress/wordpress-develop/blob/22fbb4a9ae877610f35fa5908c36839d9631674c/src/wp-includes/blocks.php#L1040-L1045